src/attestation/Attestation.spec.ts

```diff
@@ -94,4 +98,58 @@ describe('Attestation', () => {
     )
     expect(await attestation.verify()).toBeFalsy()
   })
+
+  it('compresses and decompresses the attestation object', () => {
+    const attestation = Attestation.fromRequestAndPublicIdentity(
+      requestForAttestation,
+      identityAlice
+    )
+
+    const compressedAttestation: CompressedAttestation = [
+      attestation.claimHash,
+      attestation.cTypeHash,
+      attestation.owner,
+      attestation.revoked,
+      attestation.delegationId,
+    ]
+
+    expect(compressAttestation(attestation)).toEqual(compressedAttestation)
+
+    expect(decompressAttestation(compressedAttestation)).toEqual(attestation)
+
+    expect(Attestation.decompress(compressedAttestation)).toEqual(attestation)
+
+    expect(attestation.compress()).toEqual(compressedAttestation)
+  })
+
+  it('Negative test for compresses and decompresses the attestation object', () => {
+    const attestation = Attestation.fromRequestAndPublicIdentity(
+      requestForAttestation,
+      identityAlice
+    )
+
+    const compressedAttestation: CompressedAttestation = [
+      attestation.claimHash,
+      attestation.cTypeHash,
+      attestation.owner,
+      attestation.revoked,
+      attestation.delegationId,
+    ]
+    compressedAttestation.pop()
+    delete attestation.claimHash
+
```

Comment:
With the Negative test, I have added the lines 138, 139 to change the structure of the data.